.travis.yml

```diff
@@ -6,9 +6,9 @@ git:
 dist: bionic
 
 php:
-  - 7.2
```

Comment:
Why dropping PHP 72?
I dont think we should do that at this point if we dont have to.
Rather remove 7.3, so the amount of jobs stays the same.

Reply:
Done. Your right, despite on official 7.2 EOL, users are still coupled on this version. Also, PHPUnit 9 is strict >=7.3, so added 8||9 as you mentioned before too.